posthog/cdp/templates/activecampaign/test_template_activecampaign.py

```diff
@@ -0,0 +1,56 @@
+from inline_snapshot import snapshot
+from posthog.cdp.templates.helpers import BaseHogFunctionTemplateTest
+from posthog.cdp.templates.activecampaign.template_activecampaign import (
+    template as template_activecampaign,
+)
+
+
+def create_inputs(**kwargs):
+    inputs = {
+        "accountName": "posthog",
+        "apiKey": "2f30ab262aa2bc92cc235ca437fc635dac37040d134d2d0c1be1ae0ec6a5b88476ed17b2",
```

Comment:
Every time I see a key like this I worry that it is a real one. Can you instead use a placeholder that is obviously not real like just "API_KEY" please

Reply:
```suggestion
        "apiKey": "API_KEY",
```